Update the tools part in roadmap.md (#5799)
pingcap/docs: roadmap.md

- [ ] Change column types [#17526](https://github.com/pingcap/tidb/issues/17526)
- [ ] Easier to discover performance issues and diagnose the causes [#18867](https://github.com/pingcap/tidb/issues/18867)
- [x] Support the collations of `utf8mb4_unicode_ci` and `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
- [ ] Data import
    - [ ] TiDB Lightning supports detecting conflicts in the imported data.
    - [ ] TiDB Lightning supports distributed parallel import.

> - [ ] Data replication: TiCDC is compatible with features related to TiDB Binlog, reducing the cost of TiDB Binlog users using TiCDC [#690](https://github.com/pingcap/ticdc/issues/690)

Reviewer: ```suggestion
- [ ] Data replication: TiCDC is compatible with TiDB Binlog features, which makes using TiCDC easier for TiDB Binlog users [#690](https://github.com/pingcap/ticdc/issues/690)
```

>     - [ ] Support exporting data to file systems and cloud storage (S3/GCS)

Reviewer: ```suggestion
    - [ ] Support exporting data to file systems and cloud storages (S3/GCS)
```

- [ ] Data disaster recovery: Support snapshot-level consistent data replication in the disaster recovery [#691](https://github.com/pingcap/ticdc/issues/691)

>     - [ ] Support for the TiDB disaster recovery cluster to replicate the backup cluster to a consistent state at the snapshot level when a disaster occurs in the primary cluster.

Reviewer: ```suggestion
    - [ ] The TiDB disaster recovery cluster supports replicating the backup cluster to a consistent state at the snapshot level when a disaster occurs in the main cluster
```

- [ ] Manage TiCDC using API [#736](https://github.com/pingcap/ticdc/issues/736)

> - [x] Data sharing: TiCDC supports Avro receivers, making TiCDC compatible with Kafka connect [#660](https://github.com/pingcap/ticdc/issues/660)

Reviewer: ```suggestion
- [x] Data sharing: TiCDC supports the Avro sink, which makes TiCDC compatible with Kafka connect [#660](https://github.com/pingcap/ticdc/issues/660)
```
